build.sc

```diff
@@ -58,6 +58,14 @@ trait ITestCross extends MillIntegrationTestModule with Cross.Module[String] {
       TestInvocation.Targets(Seq("__.fix")),
       TestInvocation.Targets(Seq("verify"))
     ),
+    PathRef(sources().head.path / "fix-2.12") -> Seq(
+      TestInvocation.Targets(Seq("__.fix")),
+      TestInvocation.Targets(Seq("verify"))
+    ),
```

Comment:
I just saw tests were failing because of that boyscout-rule addition, independently of the actual purpose of this PR 🤔

Reply:
The tests were not running. I could do this in a separate PR but honestly I don't see the reason.